### ISSUE TITLE
Provide binary version ?

### ISSUE THREAD

**Issue by bmillemathias** [Dev25/mcrouter_exporter#25]
as other exporters that would be nice to build binaries (I'm interested in linux x86-64) for people which does not use containers (yet ?)

best.

**Comment by smarakdas314** [comment]
Yeah could you just extract built version from docker container and put it as binary package to github, so others could download it.
